api/v2/Routes/students/projections/index.js

```diff
@@ -8,24 +8,42 @@ const router = Router({ mergeParams: true });
 
 router.get('/', async (req, res) => {
     const { email } = req.params;
-    console.debug(email);
-    const maxPoints = await getTotalPossibleScore();
-    const maxScores = await getMaxScores();
+    let maxPoints;
+    try {
+        maxPoints = await getTotalPossibleScore();
+    } catch (error) {
+        return res.status(500).json({ message: "Bins not found." });
+    }
+    let maxScores;
+    try {
+        maxScores = await getMaxScores();
+    } catch (error) {
+        return res.status(500).json({ message: "Max scores not found." });
+    }
     let studentTotalScore;
     let userGrades;
     if (isAdmin(email)) {
```

Comment:
invert this statement and return early if the else statement is longer than the body and you return right after